dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -819,7 +819,35 @@ ShowStatusPropertyDialog(
         else if (pProperties->Status == NCS_MEDIA_DISCONNECTED || pProperties->Status == NCS_DISCONNECTED ||
                  pProperties->Status == NCS_HARDWARE_DISABLED)
         {
-            ShowNetConnectionProperties(pContext->pNet, pContext->hwndDlg);
+            WCHAR buffer[100];
+            
+            LoadStringW(netshell_hInstance, IDS_NETWORKCONNECTION, buffer, _countof(buffer));
+            HWND hWndProp = FindWindow(NULL, buffer);
+
+            /* If the window is already opened, prevent it from opening again */
+            if (hWndProp != NULL)
```

Comment:
```suggestion
            /* If the window is already opened, prevent it from opening again */
            LoadStringW(netshell_hInstance, IDS_NETWORKCONNECTION, buffer, _countof(buffer));
            HWND hWndProp = FindWindow(NULL, buffer);
            if (hWndProp)
```

Reply:
> If the window is already opened, prevent it from opening again

Granted, the way it is currently done is sensitive to the current UI language being used.

Reply:
I think shell32 is supposed to check IShellWindows to see if the PIDL is already open somewhere when the reuse windows option is on. I don't think shdocvw implements the window list yet so the hack this PR is using is good enough for now.